frontend/src/styles/richTextEditor.css

```
@@ -0,0 +1,199 @@
+  background: var(--dash-outline-variant, #c3c6d7);
+  flex-shrink: 0;
+}
+
+.rte-group-legend {
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)